### PR TITLE
Fix typos and improve language

### PR DESCRIPTION
**Pull request by preciz**
Thank you for reviewing.

### CODE REVIEW

File: Fixed SEARCH/REPLACE block for `lib/tesla/middleware/path_params.ex
Comment (yordis) [inline Fixed SEARCH/REPLACE block for `lib/tesla/middleware/path_params.ex:1]: I do not understand the changes here 🤔

**Review by yordis — CHANGES_REQUESTED** [review]

File: Fixed SEARCH/REPLACE block for `lib/tesla/middleware/path_params.ex
Comment (preciz) [reply Fixed SEARCH/REPLACE block for `lib/tesla/middleware/path_params.ex:1]: Sorry, accidentally committed that. Generated by the aider-chat tool.

File: lib/tesla/adapter/mint.ex [hunk 1]
```diff
@@ -315,7 +315,7 @@ if Code.ensure_loaded?(Mint.HTTP) do
 
         {:error, _conn, error, _res} ->
           if opts[:close_conn], do: {:ok, _conn} = close(conn)
-          {:error, "Encounter Mint error #{inspect(error)}"}
+          {:error, "Encountered Mint error #{inspect(error)}"}
```
Comment (yordis) [inline lib/tesla/adapter/mint.ex:318]: I am unsure if we should change the error, even when it is a misspelling; I am not sure who would be relying on it ...

Comment (preciz) [reply lib/tesla/adapter/mint.ex:318]: Or maybe not 0 https://github.com/Logflare/logflare/blob/dc9b17541eab4a7118cdc79adaf190b48cecdca3/lib/logflare_web/live/vercellogdrains_live/index.ex#L93

Comment (yordis) [reply lib/tesla/adapter/mint.ex:318]: Do you mind adding a TODO comment to change when we decide to break change?

Comment (preciz) [reply lib/tesla/adapter/mint.ex]: Very good idea, done that

**Review by yordis — APPROVED** [review]

**Pull request merged**